packages/mesa.rb

```diff
@@ -39,6 +39,7 @@ class Mesa < Meson
   depends_on 'libxshmfence' # R
   depends_on 'libxv' => :build
   depends_on 'libxxf86vm' # R
+  depends_on 'llvm20_lib' # R
```

Comment:
Abstract with `depends_on 'llvm_lib' # R`?

Reply:
I need to modify the getrealdeps logic to make that happen ... On the list....